<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModuleVolumeWidget</class>
 <widget class="QWidget" name="ModuleVolumeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>247</width>
    <height>286</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Blending Mode</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="cbBlending"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Interpolation</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cbInterpolation"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="cbGradientOpac">
     <property name="text">
      <string>Enable Gradient Opacity</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbLighting">
     <property name="title">
      <string>Lighting</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="1">
       <widget class="QSlider" name="sliAmbient">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSlider" name="sliDiffuse">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSlider" name="sliSpecular">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Ambient</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Diffuse</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Specular</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Sp. Power</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSlider" name="sliSpecularPower">
        <property name="maximum">
         <number>200</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbJittering">
     <property name="text">
      <string>Ray jittering</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
